src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -545,5 +545,155 @@ private static DeclarationModifiers GetOverrideModifiers(ISymbol symbol)
                         : ImmutableArray.Create(codeFactory.ReturnStatement(body)));
             }
         }
+
```

Comment:
moved this logic to a common location as we will use it for Implement-Abstract-Class in https://github.com/dotnet/roslyn/pull/41730

Reply:
We should eventually move this file to `WorkspaceExtensions` shared project.